style.css

```diff
@@ -356,3 +357,238 @@
   align-self: center;
   margin-bottom: 10px;
 }
+
+/* Not display on mobile */
+.portfolio-menu {
+  display: none;
+}
+
+/* Not display on mobile */
+
+@media screen and (min-width: 1280px) {
+  * {
+    margin: 0;
+
+    /* border: 1px solid red; */
+  }
+
+  /* Content of the page */
+
+  .header-container {
+    height: 72px;
+    left: 0;
+    top: 0;
+    padding: 5px 120px 5px 150px;
+  }
+
+  .portfolio-menu,
+  a {
+    display: flex;
+    color: white;
+    list-style: none;
+    align-items: center;
+    gap: 30px;
+    text-decoration: none;
+  }
+
+  .headline-container {
+    height: 72vh;
+    background-image: url(assets/images/Illustration-header\ \(1\) .png);
+    background-position: right bottom 30%;
+    display: grid;
+    grid-template-columns: 48px 10px;
+    grid-template-rows: 200px;
+    row-gap: 10px;
+  }
+
+  .headline-text {
+    grid-column: 2/3;
+    grid-row: 1/2;
+    width: 400px;
+  }
+
+  .headline-p {
+    grid-column: 2/3;
+    grid-row: 2/3;
+    inline-size: 450px;
+    overflow-wrap: break-word;
+  }
+
+  .social-media-icones-container {
+    grid-column: 1/2;
+    grid-row: 1/3;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: fit-content;
+  }
+
+  /* Works container */
+
+  #works-container {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    grid-template-rows: auto auto auto auto;
+    row-gap: 30px;
+    padding-left: 50px;
+    padding-right: 50px;
+    margin-bottom: 100px;
+  }
+
+  .recent-works-title-block {
+    grid-column: 1 / 4;
+    grid-row: 1 /2;
+    display: flex;
+    flex-direction: row;
+    text-align: right;
+  }
+
+  .recent-works-title {
+    margin-left: 20px;
+  }
+
+  .line {
+    width: 65%;
+  }
+
+  .work-img {
+    width: 600px;
+    height: auto;
+  }
+
+  .recent-works {
+    grid-column: 1/4;
+    grid-row: 2/3;
+    display: grid;
+    grid-template-columns: auto auto;
+  }
```

Comment:
- [ ] Dear @aubinleyoung, Although your feature project is looking good, One thing missing is you need to align the image and the text properly. Please make sure The featured project image takes 60% of the page's width and is aligned to the left, and the description takes the remaining 40% and is aligned to the right. 

### Figma design 
![sdfsdf](https://user-images.githubusercontent.com/69077061/206688467-715ec125-87e4-4739-8742-b8c00e29939f.PNG)

### Your dsign
![Capture](https://user-images.githubusercontent.com/69077061/206688455-dd57ccf7-875a-466b-8de1-f7e49d8d3419.PNG)